nexusdialog/src/main/java/com/github/dkharrat/nexusdialog/controllers/LabeledFieldController.java

```
@@ -130,4 +133,10 @@ protected View createView() {
 
         return view;
     }
-}
\ No newline at end of file
+
+    @Override
+    public void setError(String message) {
```

How about also accepting `null` to be able to hide the error message?